style.css

```diff
@@ -13,7 +13,6 @@ header {
   background-color: #4caf50;
   color: white;
 }
-
```

Comment:
I think it would be better to empty the line before at-rule

Reply:
- [ ] Kindly make sure you always explain why a change was suggested. And also you may consider providing some online resources for further documentation :smiley: